include/alpaka/core/Cuda.hpp

```diff
@@ -177,13 +181,15 @@ namespace alpaka
                 if(error != CUDA_SUCCESS)
                 {
                     std::string const sError(std::to_string(file) + "(" + std::to_string(line) + ") '" + std::to_string(cmd) + "' returned error: '" + std::to_string(error) + "' (possibly from a previous CUDA call)!");
+#if ALPAKA_DEBUG != ALPAKA_DEBUG_DISABLED
                     std::cerr << sError << std::endl;
-                    ALPAKA_DEBUG_BREAK;
-                    throw std::runtime_error(sError);
-                }
-            }
-        }
-    }
+#endif
+                  ALPAKA_DEBUG_BREAK;
+                  throw std::runtime_error(sError);
+              }
+           }
+       }
+   }
```

Comment:
This indention was wrong before. I fixed that the brackets are now well aligned but not 4 spaces indented.
Should I remove the changes and keep the wrong indention or should I fix it fully?